### PR TITLE
Build and release process enhancements

### PR DESCRIPTION
**Pull request by vojtechtrefny**
I plan to release copr-builder on PyPI, we need few small changes for that.

### CODE REVIEW

**Comment by vojtechtrefny** [comment]
copr-builder on TestPyPI: https://test.pypi.org/project/copr-builder/

File: Makefile [hunk 1]
```diff
@@ -9,6 +12,23 @@ pep8:
...
+
+release: tag archive
+
+archive:
+	@git archive --format=tar --prefix=$(APPNAME)-$(VERSION)/ $(VERSION) | tar -xf -
+	@( cd $(APPNAME)-$(VERSION) && python3 setup.py -q sdist --dist-dir .. )
+	@echo "The archive is in $(APPNAME)-$(VERSION).tar.gz"
+
+bumpver:
+	@NEWSUBVER=$$((`echo $(VERSION) |cut -d . -f 2` + 1)) ; \
```
Comment (jkonecny12) [inline Makefile:28]: I guess you want to have it consistent and add space before `|`.

**Review by jkonecny12 — APPROVED** [review]
Looks good to me.

**Comment by jkonecny12** [comment]
You should also change README from `copr` to `Fedora copr` to make it obvious. Ideally having there a link.

**Comment by vojtechtrefny** [comment]
> You should also change README from `copr` to `Fedora copr` to make it obvious. Ideally having there a link.

Good idea, I reworded the description a little and added link to the Copr website.

**Review by jkonecny12 — APPROVED** [review]
LGTM!

**Pull request merged**